packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -295,6 +308,19 @@ export abstract class QueueProcessingServiceBase extends Construct {
    */
   public readonly scalingSteps: ScalingInterval[];
 
+  /**
+   * Grace period after scaling activity in seconds.
+   *
+   * Subsequent scale outs during the cooldown period are squashed so that only
+   * the biggest scale out happens.
+   *
+   * Subsequent scale ins during the cooldown period are ignored.
+   *
+   * @see https://docs.aws.amazon.com/autoscaling/application/APIReference/API_StepScalingPolicyConfiguration.html
+   * @default No cooldown
+   */
+  private readonly cooldown?: Duration;
```

Comment:
docstring not needed for a private prop right? seems like overkill. if you keep it in, the default is not 'No cooldown'

Reply:
Going to keep the docstring but shorten it to match the other private properties.